src/pipx/util.py

```diff
@@ -127,3 +128,17 @@ def run(cmd: Sequence[Union[str, Path]], check=True) -> int:
     if check and returncode:
         raise PipxError(f"{cmd_str!r} failed")
     return returncode
+
+
+def valid_pypi_name(package_name: str) -> bool:
+    # https://www.python.org/dev/peps/pep-0508/#names
+    return bool(
+        re.search(r"^([A-Z0-9]|[A-Z0-9][A-Z0-9._-]*[A-Z0-9])$", package_name, re.I)
+    )
+
+
+def full_package_description(package, package_spec):
+    name_str = f"{package}"
```

Comment:
Why is this necessary?

Reply:
Ah you're right, I don't need the f-string if that's what you're referring to.  It fell out in the refactoring that way.

Reply:
I simplified the function full_package_description and removed this.